contrib/autocomplete/bash_autocomplete

```diff
@@ -6,7 +6,7 @@ _cli_bash_autocomplete() {
      local cur opts base
```

Comment:
This file was for the old CLI lib so I think you can remove it.

https://github.com/urfave/cli#distribution